docs/src/app/components/pages/components/DropDownMenu/ExampleLongMenu.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import DropDownMenu from 'material-ui/lib/DropDownMenu';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+const longMenuItems = [];
+for (let i = 0; i < 100; i++ ) {
+  longMenuItems.push({
+    value: i,
+    text: `Item ${i}`,
+  });
+}
+
+export default class DropDownMenuLongMenuExample extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {value: 10};
+  }
+
+  handleChange = (e, index, value) => this.setState({value});
+
+  render() {
+
+    const items = longMenuItems.map((item, index) => (
```

Comment:
this isn't what makes it slow, it's the rendering. this extra work is pure js, and the iterations are only 100, this won't make much different, yet I think this is more expressive.